pkgs/build-support/docker/default.nix

```diff
@@ -856,6 +856,11 @@ rec {
     , # Time of creation of the image. Passing "now" will make the
       # created date be the time of building.
       created ? "1970-01-01T00:00:01Z"
+    , # Credentials for file ownership.
```

Comment:
The real documentation is over at `doc/build-helpers/images/dockertools.section.md`.
(I think we should remove these comments and refer to that file instead.)

Which files are affected by this?

Probably the answer also answers:
Does it interact with `fakeRootCommands`?